dd-java-agent/src/main/java/datadog/trace/bootstrap/SystemUtils.java

```diff
@@ -0,0 +1,33 @@
+package datadog.trace.bootstrap;
+
+public class SystemUtils {
```

Comment:
```suggestion
public final class SystemUtils {
```
Utils class with private constructor